components/centraldashboard-angular/frontend/package.json

```diff
@@ -6,7 +6,12 @@
     "start": "ng serve --proxy-config proxy.conf.json",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "test:prod": "ng test --no-watch --browsers ChromeHeadlessNoSandbox",
+    "tslint": "tslint -c tslint.json -p tsconfig.json",
```

Comment:
Why do we introduce rules for `tslint`, which is getting deprecated?

Let's instead have the following 2 rules:
```json
    "lint-check": "ng lint",
    "lint": "ng lint --fix",
```
similarly to KWA https://github.com/kubeflow/katib/blob/master/pkg/new-ui/v1beta1/frontend/package.json#L12-L13

We might also need to make some small changes to the `angular.json`. Take a look at the previous effort for this in https://github.com/kubeflow/kubeflow/pull/6464